seed/challenges/03-back-end-development-certification/git.json

```diff
@@ -8,67 +8,27 @@
-        "Open up <a href='http://c9.io' target='_blank'>http://c9.io</a> and sign in to your account.",
-        "Click on the \"+\" icon at the top right of the c9.io page to create a new workspace.",
-        "Give your workspace a name and an optional description.",
-        "Choose Node.js in the selection area below the name field.",
-        "Click the \"Create workspace\" button.",
-        "Once C9 builds and loads your workspace, you should see a terminal window in the lower right hand corner. In this window use the following commands. You don't need to know what these mean at this point.",
-        "Install <code>git-it</code> with this command: <code>npm install -g git-it</code>",
-        "Before you start, read <a href='https://help.github.com/articles/adding-a-new-ssh-key-to-your-github-account/' target='_blank'>Adding a new SSH key to your GitHub account</a> if you have two-factor authentication set up on your GitHub account.",
-        "Now start the tutorial by running <code>git-it</code>.",
-        "Note that you can resize the c9.io's windows by dragging their borders.",
-        "Make sure that you are always in your project's \"workspace\" directory. You can always navigate back to this directory by running this command: <code>cd ~/workspace</code>.",
-        "You can view this Node School module's source code on GitHub at <a href='https://github.com/jlord/git-it' target='_blank'>https://github.com/jlord/git-it</a>.",
-        "Complete \"Get Git\"",
-        "Complete \"Repository\"",
-        "Complete \"Commit to it\"",
-        "Complete \"Githubbin\"",
-        "Complete \"Remote Control\"",
-        "Complete \"Forks and Clones\"",
-        "Complete \"Branches aren't just for Birds\"",
-        "Complete \"It's a Small World\"",
-        "Complete \"Pull, Never out of Date\"",
-        "Complete \"Requesting you Pull, Please\"",
-        "Complete \"Merge Tada!\"",
+        "To complete this challenge, you will need to install a few things on your computer - a text editor, Git, and the Git-It-Electron desktop app.",
+        "Text Editor - A text editor is a good tool to have as a programmer. You can download and install <a href=\"https://atom.io/\" target=\"_blank\">Atom</a>, <a href=\"https://www.sublimetext.com/3\" target=\"_blank\">Sublime Text</a>, or <a href=\"http://brackets.io/\" target=\"_blank\">Brackets</a>. All these editors are available for Windows, Mac and Linux platforms. Alternatively, if you already have a text editor, or you would like to use a different one, you're welcome to do so.",
+        "Git - Download and install Git from <a href=\"https://git-scm.com/download/\" target=\"_blank\">git-scm</a>. You can complete the installation with the recommended defaults. Note - for Windows users - as the installation completes, on the final screen, click on \"Launch Git Bash\". Mac and Linux users may use their terminal instead.",
```

Comment:
> for example for Debian-based distros like Ubuntu just use `sudo apt-get install git`

I don't this is necessary, getting them via apt packaging is a super user step, which most super users would be aware already.

This would make sense to newbies only if we go ahead and add pointers for the whats apt packaging.

Again that's beyond the scope of this challenge, I believe we should rather leave this as it is. The git-scm site documents enough for installing the binaries.